app/authoring/user-interactions.md

```diff
@@ -23,13 +23,19 @@ The `prompt` method is asynchronous and return a promise. You'll need to return
 ```js
 module.exports = generators.Base.extend({
   prompting: function () {
-    return this.prompt({
+    return this.prompt([{
       type    : 'input',
       name    : 'name',
       message : 'Your project name',
       default : this.appname // Default to current folder name
-    }).then(function (answers) {
-      this.log(answers.name);
+    }, {
+      type    : 'confirm',
+      name    : 'cool',
+      message : 'Would you like to enable the Cool feature?',
+      default : this.cool
```

Comment:
I'm all for two question, let's just not add a `default` to the second one as `this.cool` doesn't exists by default.